kedro/framework/cli/cli.py

```diff
@@ -38,6 +39,13 @@
 v{version}
 """
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.ERROR)
```

Comment:
This will hide `WARNING` level logs as I understand, what's the reasoning behind this?

Reply:
My initial idea was that, since this would happen instead of the run stopping for an exception, it should appear only when an error occurs. But yeah it makes sense, we can lower it to a warning level.